tagging/ebs/es.go

```diff
@@ -0,0 +1,70 @@
+//   Copyright 2020 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package taggingebs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/olivere/elastic"
+
+	"github.com/trackit/trackit/aws"
+	indexSource "github.com/trackit/trackit/aws/usageReports/ebs"
+	"github.com/trackit/trackit/es"
+)
+
+func fetchReports(ctx context.Context, awsAccount aws.AwsAccount) ([]*elastic.SearchHit, error) {
+	client := es.Client
+	indexName := es.IndexNameForUserId(awsAccount.UserId, indexSource.IndexPrefixEBSReport)
+
+	indexExists, err := client.IndexExists(indexName).Do(ctx)
+	if err != nil {
+		return nil, err
+	}
+	if !indexExists {
+		return []*elastic.SearchHit{}, nil
+	}
+
+	res, err := queryEs(ctx, indexName, awsAccount)
+	if err != nil {
+		return nil, err
+	}
+
+	return processSearchResult(res)
+}
+
+func queryEs(ctx context.Context, indexName string, awsAccount aws.AwsAccount) (*elastic.SearchResult, error) {
+	client := es.Client
+
+	index := client.Search().Index(indexName)
+	topHitsAggregation := elastic.NewTopHitsAggregation().Size(2147483647).FetchSourceContext(elastic.NewFetchSourceContext(true).Include("account", "snapshot.id", "snapshot.region", "snapshot.tags"))
```

Comment:
use the format with only one variable to be consistent with the rest of the code